README.md

```diff
@@ -45,28 +37,42 @@ Available generators:
 
 **Note: Generators are to be run from the root directory of your app.**
```

Comment:
This might be a little confusing from the start for a new user. Since they wouldn't have an app folder yet. Maybe make this note about the Sub-Generators? Or clarify where this should be ran from, to generate the actual app for the first time?